sources/core/Stride.Core/Stride.Core.csproj

```diff
@@ -16,6 +16,11 @@
     <StrideBuildTags>*</StrideBuildTags>
     <RestorePackages>true</RestorePackages>
     <ExtrasUwpMetaPackageVersion>6.2.12</ExtrasUwpMetaPackageVersion>
+    <LangVersion>12.0</LangVersion>
```

Comment:
Is this still necessary?
cf https://github.com/stride3d/stride/pull/1616#discussion_r1372368581